src/main/kotlin/app/revanced/patches/youtube/utils/settings/VisualPreferencesIconsPatch.kt

```diff
@@ -0,0 +1,380 @@
+                "revanced_hide_quick_actions_save_to_playlist_button" -> "revanced_hide_playlist_button_icon"
+                "revanced_hide_quick_actions_share_button" -> "revanced_hide_shorts_share_button_icon"
+                "revanced_hide_remix_button" -> "revanced_hide_shorts_remix_button_icon"
+                "revanced_hide_share_button" -> "revanced_hide_shorts_share_button_icon"
+                "revanced_hide_shorts_comments_button" -> "revanced_hide_quick_actions_comment_button_icon"
+                "revanced_hide_shorts_dislike_button" -> "revanced_preference_screen_ryd_icon"
+                "revanced_hide_shorts_like_button" -> "revanced_hide_quick_actions_like_button_icon"
+                "revanced_hide_shorts_navigation_bar" -> "revanced_preference_screen_navigation_buttons_icon"
+                "revanced_hide_shorts_shelf_home_related_videos" -> "revanced_hide_navigation_home_button_icon"
+                "revanced_hide_shorts_shelf_subscriptions" -> "revanced_hide_navigation_subscriptions_button_icon"
+                "revanced_hide_shorts_toolbar" -> "revanced_preference_screen_toolbar_icon"
+                "revanced_preference_screen_account_menu" -> "account_switcher_key_icon"
+                "revanced_preference_screen_channel_bar" -> "account_switcher_key_icon"
+                "revanced_preference_screen_channel_profile" -> "account_switcher_key_icon"
+                "revanced_preference_screen_comments" -> "revanced_hide_quick_actions_comment_button_icon"
+                "revanced_preference_screen_feed_flyout_menu" -> "revanced_preference_screen_player_flyout_menu_icon"
+                "revanced_preference_screen_haptic_feedback" -> "revanced_enable_swipe_haptic_feedback_icon"
+                "revanced_preference_screen_patch_information" -> "about_key_icon"
+                "revanced_preference_screen_shorts_player" -> "revanced_preference_screen_shorts_icon"
+                "revanced_preference_screen_video_filter" -> "revanced_preference_screen_video_icon"
+                "revanced_swipe_gestures_lock_mode" -> "revanced_hide_player_flyout_menu_lock_screen_icon"
+                "revanced_disable_hdr_auto_brightness" -> "revanced_disable_hdr_video_icon"
+                else -> "${title}_icon"
+            }
+        }
+
+        val validExtendedBrandIcon = validExtendedBrand.associateWith { "${it}_icon" }
+
+        // Copy resources
+        val emptyIcon = "empty_icon"
+        val resourcesToCopy = mutableListOf(
+            ResourceGroup("drawable-xxhdpi", "$emptyIcon.png"),
+            ResourceGroup("drawable", *validTitlesIcons.values.map { "$it.xml" }.toTypedArray())
+        )
+
+        fun copyResourcesWithFallback(iconPath: String) {
+            try {
+                context.copyResources(
+                    iconPath,
+                    ResourceGroup("drawable", "revanced_extended_settings_key_icon.xml")
```

Comment:
`ResourceGroup("drawable", "revanced_extended_settings_key_icon.xml")` is used in two places, so you can declare it as a variable